src/db/commit.rs

```diff
@@ -183,7 +220,15 @@ impl Commit {
         let commit = commit_fb::Commit::create(&mut builder, commit_args);
         builder.finish(commit, None);
 
-        let chunk = dag::Chunk::new(builder.collapse(), &[value_hash]);
+        let mut refs = vec![value_hash];
+        if let Ref::Strong(basis_hash) = basis_hash {
```

Comment:
It was my understanding that basis was only going to be strong for local commits?

Reply:
Yup, `new_impl` is internally used by `new_local` and `new_snapshot`